e2e/certificates/ca.crt

```diff
@@ -0,0 +1,30 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I would not commit these here but rather generate them each time the test runs and delete them afterwards. It's often easier to commit them, but at the same time, I would prefer to avoid having any secret keys or certificates even for testing committed to prevent any possibility someone might accidentally use them somewhere.

Reply:
@tanertopal wdyt, should we also move the `certificates` directory into the `https-bare` dir? They are only required for that single test

Reply:
If they are only required here, then let's move it. The directory could also contain the script to generate the certificates.